README.md

```diff
@@ -17,18 +18,23 @@ provider:
   webhook:
     image:
       repository: ghcr.io/inovex/external-dns-openstack-webhook
-      tag: 1.0.0
+      tag: 1.1.0
     extraVolumeMounts:
       - name: oscloudsyaml
         mountPath: /etc/openstack/
     resources: {}
-    securityContext:
-      runAsUser: 1000
+extraVolumes:
+  - name: oscloudsyaml
+    secret:
+      secretName: oscloudsyaml
 ```
 
-The referenced `extraVolumeMount` points to a `Secret` containing the `clouds.yaml` file, which provides the OpenStack Keystone credentials to the webhook provider. While it seems cumbersome to require a file instead of the commonly used `OS_*` environment variables, the use of a `clouds.yaml` file offers more structure, capabilities and allows for better validation.
+The referenced `extraVolumeMount` points to a `Secret` containing a [`clouds.yaml` file](https://docs.openstack.org/python-openstackclient/latest/configuration/index.html#clouds-yaml),
+which provides the OpenStack Keystone credentials to the webhook provider.
+`OS_*` environment variables are not supported for configuration, since the use of a `clouds.yaml` file offers more structure, capabilities and allows for better validation.
+The one exception to this is `OS_CLOUD` for setting the name of the cloud in `clouds.yaml` to use.
```

Comment:
We default to `OS_CLOUD=openstack` (https://github.com/inovex/external-dns-openstack-webhook/blob/82502b74af773e0826b8a50b371c98c2d207932c/goreleaser.dockerfile#L8), maybe that's worth mentioning, so it's not actually required to set anything if you name your cloud "openstack".

Reply:
This is already mentioned in line 37